package.json

```diff
@@ -19,6 +19,8 @@
     "npm": ">=7.0.0"
   },
   "dependencies": {
+    "@wordpress/edit-post": "^7.20.1",
+    "@wordpress/plugins": "^6.11.1",
```

Comment:
I raised the discussion on Slack (thread: p1697484876676319-slack-C036R3Y5ET0), but looking at `package.json`, we actually might be able to remove those, despite what I said previously. 

When running `npm install` here, I see a ton of warnings, which..isn't very good either. 

Sorry for the annoyance!